tests/test_authentication.py

```diff
@@ -67,25 +71,31 @@ def test_loading_user_does_not_parse_standard_claims(jwt_partial_payload: dict)
         "auth_time": current_ts,
     }
 
-    id_token = encode_token(
+    id_token = encode_jwt(
         {
             "cognito:username": str(uuid4()),
             "custom:id": str(uuid4()),
             **standard_claims,
-        }
+        },
+        SAMPLE_PRIVATE_KEY,
     )
     user = User(id_token)
     for key in standard_claims:
         assert not hasattr(user, key)
 
 
-def test_user_raises_when_more_attributes_than_1000() -> None:
+def test_user_raises_when_more_attributes_than_1000(
+    allowed_audiences: list[str],
+) -> None:
```

Comment:
I don't get what happened with your auto-formatting? This line shouldn't be changed this way 🤔 
```suggestion
def test_user_raises_when_more_attributes_than_1000(allowed_audiences: list[str]) -> None:
```